README.md

```diff
@@ -2,379 +2,101 @@
 
 [![CI](https://github.com/kyazdani42/nvim-tree.lua/actions/workflows/ci.yml/badge.svg)](https://github.com/kyazdani42/nvim-tree.lua/actions/workflows/ci.yml)
 
-## Notice
-
-This plugin requires [neovim >=0.7.0](https://github.com/neovim/neovim/wiki/Installing-Neovim).
-
-If you have issues since the recent setup migration, check out [this guide](https://github.com/kyazdani42/nvim-tree.lua/issues/674)
+<add more info here + feature list>
```

Comment:
I don't quite know what to put in here. Maybe we don't need anything.

Reply:
I was thinking about some high level list of what that plugin does. :)